types/iyzipay/index.d.ts

```diff
@@ -0,0 +1,1601 @@
+/* eslint-disable @typescript-eslint/naming-convention */
```

Comment:
Please remove those exclusion (why there are there in the first place?)

Reply:
I have removed. I was using another naming convention however I updated the type and interface declarations accordingly.